scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua

```diff
@@ -57,7 +75,7 @@ entity.onSpellPrecast = function(mob, spell)
         spell:setFlag(xi.magic.spellFlag.HIT_ALL)
         spell:setRadius(30)
         spell:setAnimation(280)
-        spell:setMPCost(1)
+        spell:setMPCost(0)
```

Comment:
Oh, sneaky...

Unrelated to this line, doesnt KB have a draw-in mechanic or something?

I'm sure it doesnt go into the tunnels

Reply:
It has one already on it from cocosolo's draw in rework. It has an arena that draws in if you try going into the tunnels.